.github/workflows/auto-assigner.yml

```diff
@@ -0,0 +1,14 @@
+name: Auto-assign author to pull request
+
+on:
+  pull_request:
+    types: [opened]
+
+jobs:
+  auto-assign:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Auto-assign author to PR
+        uses: hkusu/review-assign-action@v1
```

Comment:
on the topic of letting AI write code unsupervised...

```
pozil/auto-assign-issue@v2
```
☝️ i'm 95% certain that the library that was suggested initially only supports assigning _reviewers_. 🙃 

https://github.com/pozil/auto-assign-issue